packages/firebase_database/test/firebase_database_test.dart

```diff
@@ -117,6 +117,16 @@ void main() {
           ]),
         );
       });
+      test('update', () async {
+        final dynamic value = <String, dynamic>{'hello': 'world'};
+        await database.reference().child("foo").update(value);
+        expect(
+            log,
+            equals(<MethodCall>[
+              new MethodCall("DatabaseReference#update",
+                  <String, dynamic>{'path': 'foo', 'value': value})
```

Comment:
I tried my best to fix these, I hope it is good now. 

There should be a better way to do this that I am missing probably. An intellij setting, or something?